sdks/go/node/api/handler/events/stream/handler.go

```diff
@@ -59,7 +59,7 @@ func (hdlr _handler) Handle(
 	if sinceString := httpReq.URL.Query().Get("since"); sinceString != "" {
 		sinceTime, err := time.Parse(time.RFC3339, sinceString)
 		if err != nil {
-			http.Error(httpResp, err.Error(), http.StatusBadRequest)
```

Comment:
Are these errors going to get lost?

Reply:
They never were used; the connection is upgraded to a websocket above so these all were doing nothing other than logging that we're (invalidly) attempting to writing to hijacked connection .